### ISSUE TITLE
Keyboard fails to access Year on Focus

### ISSUE THREAD

**Issue by NagendraReddyDotula** [dmtrKovalenko/material-ui-pickers#951]
When I am trying to access the Year using keyboard I am not getting focus on the Year In the Calendar. For Other fields Focus is appearing except Year. Can you please provide the focus on Year also..

**Comment by diogoweb2** [comment]
I have the same issue. (it's a WCAG issue to not offer keyboard navigation to YEAR)

**Comment by sperry94** [comment]
Are you referring to the Year selection in the toolbar, like below? If so, this is being addressed in #976 for all picker toolbars.

<img width="132" alt="image" src="https://user-images.githubusercontent.com/10750197/55666895-4615cc00-5812-11e9-9e68-378b6036f1b9.png">
